Source/Fuse.Scripting.JavaScript/JavaScriptCore/3rdparty/JavaScriptCore.cpp.uxl

```diff
@@ -0,0 +1,11 @@
+<Extensions Backend="CPlusPlus">
+	<CopyFile HeaderFile="JavaScriptCore/JavaScript.h" />
```

Comment:
Does this mean that we're copying these headers on iOS as well? Do we want that and not use the ones from the framework? For example the `JSTypedArray` header isn't present on some older versions.

Reply:
I think we do, and we probably shouldn't...